packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Deployment/Netlify.tsx

```diff
@@ -0,0 +1,200 @@
+import { Button } from '@codesandbox/common/lib/components/Button';
+import { resolveDirectory } from '@codesandbox/common/lib/sandbox/modules';
+import getTemplate from '@codesandbox/common/lib/templates';
+import { observer } from 'mobx-react-lite';
+import React, { useEffect, useState } from 'react';
+import LinkIcon from 'react-icons/lib/fa/external-link';
+import Cogs from 'react-icons/lib/fa/cogs';
+import LightningIcon from 'react-icons/lib/md/flash-on';
+
+import DeploymentIntegration from 'app/components/DeploymentIntegration';
+import NetlifyLogo from 'app/components/NetlifyLogo';
+import { useSignals, useStore } from 'app/store';
+import getNetlifyConfig from 'app/utils/getNetlifyConfig';
+
+import { WorkspaceInputContainer, WorkspaceSubtitle } from '../../elements';
+
+import {
+  Deploys,
+  Deploy,
+  Name,
+  Link,
+  DeploysWrapper,
+  Wrapper,
+  ButtonContainer,
+} from './elements';
+
+const getFunctionDir = sandbox => {
+  try {
+    return resolveDirectory(
+      getNetlifyConfig(sandbox).functions,
+      sandbox.modules,
+      sandbox.directories
+    );
+  } catch (e) {
+    return [];
+  }
+};
+
+export const Netlify = observer(() => {
+  const {
+    deployment: { getNetlifyDeploys, deployWithNetlify },
+    modalOpened,
+  } = useSignals();
+  const { deployment, editor } = useStore();
+
+  const [isVisible, setVisibility] = useState(false);
+
+  useEffect(() => getNetlifyDeploys(), []); // eslint-disable-line
```

Comment:
@Saeris It's a best practice to add the `getNetlifyDeploys ` function to the `useEffect` dependencies even though you only want the effect to run on mount

```suggestion
  useEffect(() => getNetlifyDeploys(), [getNetlifyDeploys]);
```

Reply:
Agree!